tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -26,6 +25,14 @@ Verify Pachyderm Can Be Installed Using OpenShift Console
     Wait Until Status Is Running
     Go To RHODS Dashboard
     Verify Service Is Enabled    Pachyderm
+    Launch JupyterHub Spawner From Dashboard
+    Wait Until Page Contains Element    xpath://input[@name="Standard Data Science"]
+    Wait Until Element Is Enabled       xpath://input[@name="Standard Data Science"]    timeout=10
```

Comment:
I think these two lines are not necessary

Reply:
Updated !